src/main/java/com/google/cloud/tools/app/api/deploy/DeployConfiguration.java

```diff
@@ -28,17 +28,17 @@
 
   String getDockerBuild();
 
-  boolean isForce();
```

Comment:
and gcloud binary flags don't take values, if we want to add the no-stop-previous-version flag we can